shared/react-native/ios/KeybaseShare/ShareViewController.m

```diff
@@ -377,33 +452,65 @@ - (void)didSelectPost {
   [self showProgressView];
   for (int i = 0; i < [items count]; i++) {
     BOOL lastItem = (BOOL)(i == [items count]-1);
-    [self processItem:items[i] lastItem:lastItem];
+    if (self.convTarget) {
+      [self processChatItem:items[i] lastItem:lastItem];
+    } else {
+      [self processFilesItem:items[i] lastItem:lastItem];
+    }
   }
 }
 
 - (NSArray *)configurationItems {
-  SLComposeSheetConfigurationItem *item = [[SLComposeSheetConfigurationItem alloc] init];
-  item.title = @"Share to...";
-  item.valuePending = !self.hasInited; // show a spinner if we haven't inited
+  // Share to chat
+  SLComposeSheetConfigurationItem *chat = [[SLComposeSheetConfigurationItem alloc] init];
```

Comment:
Would you mind if I played with this a bit? I think this interface can play out a little nicer, I can commit to this branch if that is ok.

Reply:
I can also do as a followup, whatever you prefer.

Reply:
Let's do it as a followup.